package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/core-monorepo",
-  "version": "436.0.0",
+  "version": "439.0.0",
```

Comment:
Maybe you want to bump this to 438.0.0 and rename the PR to "Release 438.0.0"? We can merge this one in before the other 438.0.0 PR if it works better for you.

Reply:
Fixed in https://github.com/MetaMask/core/pull/5978/commits/3f677e84325a5e00277c0868d5fe948d04aa9330